README.rst

```diff
@@ -68,6 +68,17 @@ Returns ``True`` if all elements of sequence are equals:
    True
 
 
+Function ``=if``
+''''''''''''''''
+
+Evaluates condition and returns first value if true, otherwise, returns second value.
+
+    >>> dicteval({"=if": [{"=": "@{var > 5}"}, "yes", "no"]}, context={"var": 6})
+    'yes'
+    >>> dicteval({"=if": [{"=": "@{var > 5}"}, "yes", "no"]}, context={"var": 4})
+    'no'
```

Comment:
I thought about that while working on this. My initial reaction was to raise an `IndexError`, but didn't explicitly because it would happen naturally right now. However, I think I personally think that returning `None` makes a bit more sense from a usability perspective.

Either way, I can update this and add a test for that case, based on whichever solution is preferred.

Reply:
I'm +1 to return None too :) Let's go in that direction? :)

Reply:
Sounds good. Updated, assuming the build works.